tests/isolate/test_cpu.py

```diff
@@ -16,7 +16,7 @@ class TestCpu(unittest.TestCase):
 
     def test_assign_one_thread_empty_cpu(self):
         """
-        Workload 0: 1 cores --> p:0 c:0 t:0
+        Workload 0: 1 thread --> (p:0 c:0 t:0)
```

Comment:
Why the change to `thread`?  This means `Workload 0` is requesting 1 thread, and it should get the thread 0 on core 0 on package 0.